CONTRIBUTING.md

```diff
@@ -71,35 +73,33 @@ type(scope): description
 
 In most cases, we use the following types:
 
-- `fix`: for anything that contains the resolution of an issue (identified or not)
-- `feat`: for anything that contains a new feature
+- `fix`: for any resolution of an issue (identified or not)
+- `feat`: for any new feature
+- `refactor`: for any code change that neither adds a feature nor fixes an issue
 - `chore`: for anything that is not related to the library itself (doc, tooling)
```

Comment:
we should probably mention `docs` separately, since this what you naturally  did in this commit message even